packages/data-explorer-ui/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@clevercanary/data-explorer-ui",
-  "version": "0.18.1",
+  "version": "0.19.0",
```

Comment:
Hi @BruceRodrigues can you revert the package bump? I will bump the versions from now on when I do the release. We can let main go on with new changes after a release without changing the version as all releases will be tagged from now on. I will add a write up for this on the repo readme.